core/src/mindustry/ui/dialogs/SettingsMenuDialog.java

```diff
@@ -25,6 +25,10 @@
 import mindustry.ui.*;
 
 import java.io.*;
+import java.time.Instant;
```

Comment:
`java.time` is a Java 8 API. Do not use it.

Reply:
linking a potential alternative @lorenuars19 can copy from that is already in use elsewhere in the mindustry codebase:

https://github.com/Anuken/Mindustry/blob/master/server/src/mindustry/server/ServerControl.java#L207

Reply:
You're still using `java.time` here, which is a Java 8 API. The server uses it, yes, but that's because the headless server code **never** runs on Android and hence always has access to this API.

Reply:
could I use [SimpleDateFormat](https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/text/SimpleDateFormat.html)

which is JAVA 15 ?